src/AccordionItemTitle/__snapshots__/accordion-item-title.spec.js.snap

```diff
@@ -0,0 +1,86 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AccordionItemTitle renders correctly when pressing another key 1`] = `
+<div
+  aria-controls=""
+  aria-expanded={false}
+  className="accordion__title"
+  id=""
+  onClick={[Function]}
+  onKeyPress={[Function]}
+  role=""
+  tabIndex="0"
+>
+  <div>
+    Fake Title
+  </div>
+</div>
+`;
+
+exports[`AccordionItemTitle renders correctly when pressing enter 1`] = `
+<div
+  aria-controls=""
+  aria-expanded={false}
+  className="accordion__title"
+  id=""
+  onClick={[Function]}
+  onKeyPress={[Function]}
+  role=""
```

Comment:
I don't think it's ok to have `onClick` and `onKeyPress` on a div if the `role` isn't set to anything meaningful. Should this be a `role="button"`? Also, is it ok for `aria-controls` to be set to nothing?